javalib/src/main/scala/java/util/Date.scala

```diff
@@ -0,0 +1,40 @@
+package java.util
+
+/** Ported from Scala JS and Apache Harmony
+ * - omits deprecated methods
+ * - immutable, setTime not implemented
+ * - toString not jdk compatible
+ */
+class Date(milliseconds: Long)
```

Comment:
We need to have either `Calendar` or `java.time` or some other equivalent to implement the deprecated methods. My view was that those methods are probably not used in any Scala API and was thinking that Date is not used for much except a container to the time in millis. We could get the deprecated methods by calling into C possibly. In ScalaJS most of them are delegation into js.Date. https://github.com/scala-js/scala-js/blob/master/javalib/src/main/scala/java/util/Date.scala